rpcs3/Emu/Cell/Modules/cellSaveData.cpp

```diff
@@ -1238,6 +1291,11 @@ static NEVER_INLINE error_code savedata_op(ppu_thread& ppu, u32 operation, u32 v
 			fmt::throw_exception("Failed to create directory %s (%s)", new_path, fs::g_tls_error);
 		}
 
+		// add file list per FS order to PARAM.SFO
+		std::string final_blist;
+		final_blist = fmt::merge(blist, "/");
+		psf.emplace("RPCS3_BLIST", psf::string(final_blist.size(), final_blist));
```

Comment:
Oh, it won't work right. Use psf::assign

Reply:
It worked fine because it doesn't contain previous content from param.sfo(it's cleared and values are inserted again). But I changed to assign anyway as it can't hurt.